src/Server/Packets/Handle/HandlePacketChatMessage.cs

```diff
@@ -15,14 +16,14 @@ public class HandlePacketChatMessage : HandlePacket
 
         public HandlePacketChatMessage() => Opcode = ClientPacketOpcode.ChatMessage;
 
-        private readonly string[] BlockedStrings = new string[] 
-        {
-            "fuck",
-            "nigga",
-            "nigger",
-            "cunt",
-            "bitch"
-        };
+        // Toggleable profanifty options
+        public bool useProfanityFilter = false;
+        public bool preventAllCaps = false;
+
+        // read in bad words from file and split them into string array
+        private readonly static string blockedStringFile = File.ReadAllText(@"\Blocked-Strings-List");
```

Comment:
I've never seen anyone use `@` symbol before, does this mean it's relative to the file this script is being executed in? I know if you removed the `@` it would be relative to `bin/Debug/net5.0`

Reply:
The @ symbol just means "Interpret this string literally" so you cant use escape characters, makes it easier to read really.
For example:
Writing the path without it would look like - string filename = "c:\\documents\\files\\u0066.txt";
Whereas writing it with the symbols is - string filename = @"c:\documents\files\u0066.txt";
But both of these both do the exact same thing